src/server/utils/executor/executor.ts

```diff
@@ -109,26 +114,26 @@ export function externalFactory(dataSource: DataSource, druidRequester: Requeste
   };
 
   if (dataSource.introspection === 'none') {
-    return Q(External.fromJS({
-      engine: 'druid',
+    return Q(new DruidExternal({
+      suppress: true,
       dataSource: dataSource.source,
       timeAttribute: dataSource.timeAttribute.name,
       customAggregations: dataSource.options['customAggregations'],
-      attributes: deduceAttributes(dataSource),
+      attributes: AttributeInfo.applyOverrides(deduceAttributes(dataSource), dataSource.attributeOverrides),
```

Comment:
not a change but was wondering why 'customAggregations' is accessed with brackets?